indico/modules/events/abstracts/__init__.py

```diff
@@ -140,14 +140,9 @@ def _reviewing_area_visible(conf):
 
     yield MenuEntryData(title=_("Book of Abstracts"), name='abstracts_book', endpoint='abstracts.export_boa',
                         position=9, visible=lambda event: event.has_feature('abstracts'), static_site=True)
-
-    yield MenuEntryData(title=_("Call for Abstracts"), name='call_for_abstracts', endpoint='event.conferenceCFA',
-                        position=2, visible=lambda event: event.has_feature('abstracts'))
-    yield MenuEntryData(title=_("My Abstracts"), name='user_abstracts', visible=_my_abstracts_visible,
```

Comment:
`_my_abstracts_visible` is now unused (and so is the `has_user_abstracts` util function, so it can go away and the `_query_user_abstracts` code can be moved into `get_user_abstracts` or maybe get rid of that one too and just add the query in `RHMyAbstractsBase` since it's only used there now)